### PR TITLE
Fix error check on SetInitialWallpaper

### PR DESCRIPTION
**Pull request by ernierasta**
Hi,
you are never checking error there, so after running loop all looks ok, even when stw file has wrong image paths.

### CODE REVIEW

**Comment by ernierasta** [comment]
The same problem is in `gnometimed`. Maybe loop related methods should be taken out of this packages? Do not know code base I may be wrong ...

File: eventloop.go [hunk 1]
```diff
@@ -249,7 +249,9 @@ func (stw *Wallpaper) EventLoop(verbose bool, setWallpaperFunc func(string) erro
 		fmt.Println("Using the Simple Timed Wallpaper format.")
 	}
 
-	stw.SetInitialWallpaper(verbose, setWallpaperFunc)
+	if err := stw.SetInitialWallpaper(verbose, setWallpaperFunc); err != nil {
+		fmt.Print(err)
```
Comment (xyproto) [inline eventloop.go:253]: Nice find! What do you think about returning the error here, instead of printing it?

**Review by xyproto — APPROVED** [review]

**Comment by xyproto** [comment]
Updated the code in `gnometimed` as well.

**Comment by xyproto** [comment]
And I agree that some of the code in `simpletimed` and `gnometimed` could be moved to for instance the [`github.com/xyproto/event`](https://github.com/xyproto/event) package, but I wanted to keep that package focused on events only, and not wallpapers. Perhaps a new package could solve this.

File: eventloop.go (hunk 1 above)
Comment (ernierasta) [reply eventloop.go:253]: Well, depends on what you want to achieve. I have an impression, that currently if one image is missing program will run anyway, just ignore this one image. If it returns here, program can work/not work depending on hour you run it. Unless all errors always end program - then you will not allow missing files. 

Not sure what is better.

**Pull request merged**